### PR TITLE
user-and-help-icons

### PR DESCRIPTION
**Pull request by jtrinklein**
### Review
 - [ ] @boskya

### After Merge

- [ ] [site](https://daptiv.github.io/style-guide) is updated

### CODE REVIEW

**Comment by jamest** [comment]
screen shot 2015-09-22 at 3 51 23 pm

File: docs/docs.scss [hunk 1]
```diff
@@ -30,11 +31,19 @@ $icon-between-spacing: 5px;
...
-    text-align: center;
+    height: 30px;
+    margin: $icon-block-margin;
+    padding: $icon-block-padding;
     width: $icon-block-size;
+    &:hover {
+        background-color: $color-secondary-light;
+    }
+    .icon {
+        margin-right: 5px;
```
Comment (boskya) [inline docs/docs.scss]: moved 5px to variable

File: docs/docs.scss [hunk 2]
```diff
@@ -30,11 +31,19 @@ $icon-between-spacing: 5px;
     border: 1px solid $color-primary-medium;
     border-radius: 3px;
     display: inline-block;
-    height: $icon-block-size;
-    line-height: $icon-block-size;
-    margin-right: $icon-between-spacing;    
-    text-align: center;
+    height: 30px;
```
Comment (boskya) [inline docs/docs.scss]: move height to variable

**Comment by boskya** [comment]
I like screenshot!

**Comment by boskya** [comment]
minor comment and then :shipit:

**Pull request merged**